### DISCUSSION TITLE
What's the best practice for setup supabase auth?

### DISCUSSION THREAD

**Discussion by darmau** [remix-run/remix#9747] [Q&A] [answered]
I'm trying build a website using Remix with Supabase Auth. But I can't find any document about what is the best way to setup Auth for Remix. 

Is there any articles or documents about this topic? Can it added into official document?

I do think Remix should focus more on official documents, provide more best practice with other products.

**Comment by sergiodxa** [comment, marked as answer]
[Supabase has docs for using it with Remix](https://supabase.com/docs/guides/auth/auth-helpers/remix).

It's impossible for a single, focused on features and bug fixing, team to write docs for integrating with every possible tool or service out there, the amount of auth providers, db providers, queues providers, etc. is too large, it's simpler to let other products write guides on how to integrate with Remix as well as how they write for Next, Svelte, Nuxt, etc.
